components/Card.js

```diff
@@ -0,0 +1,43 @@
+import { medications } from "@/lib/db";
+import styled from "styled-components";
+
+const CardContainer = styled.div`
+  border: 0px groove #ccc;
+  padding: 1.3em;
+  border-radius: 5px;
+  margin: 1.5em;
+  flex-direction: column;
+  justify-content: space-between;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.26);
+  @media (max-width: 375px) {
+    margin: 5%;
+  }
+`;
+const Title = styled.h1`
```

Comment:
`h1` ist the most important heading level in your app and should be used only once [(see MDN)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page) per page. `h2` would be the correct HTML element.